modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/ShowPartialResultsAlert.tsx

```diff
@@ -38,9 +38,13 @@ const ShowPartialResultsAlert: React.FC<IProps> = ({
 				onClose={dismissible ? () => setDismissed(true) : undefined}
 				title="Info"
 			>
-				{Liferay.Language.get(
-					'your-responses-will-be-visible-to-all-form-respondents'
-				)}
+				{dismissible
```

Comment:
It is not a `dismissible` property responsibility to select the proper messages.

IMHO, a better approach would be turn the message into a property, so that you can set it on component use:

`modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/ShowPartialResultsAlert.tsx`
```javascript
const ShowPartialResultsAlert: React.FC<IProps> = ({
	dismissible,
	message,
	showPartialResultsToRespondents,
}) => {
	...

	return (
		<div
			className={classNames('lfr-ddm__show-partial-results-alert', {
				'lfr-ddm__show-partial-results-alert--hidden': isDismissed,
			})}
		>
			<ClayAlert
				...
			>
				{message}
```
`modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp`
```jsp
<react:component
	module="admin/js/util/ShowPartialResultsAlert"
	props='<%=
		HashMapBuilder.<String, Object>put(
			"dismissible", true
		).put(
			"message", LanguageUtil.get(request, "respondents-can-see-all-submitted-form-data")
		).put(
			"showPartialResultsToRespondents", ddmFormDisplayContext.isShowPartialResultsToRespondents()
		).build()
	%>'
/>
``` 

`modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf`:
```jsp
<react:component
	module="admin/js/util/ShowPartialResultsAlert"
	props='<%=
		HashMapBuilder.<String, Object>)put(
			"message", LanguageUtil.get(request, "respondents-can-see-all-submitted-form-data")
		.put(
			"showPartialResultsToRespondents", ddmFormAdminDisplayContext.isShowPartialResultsToRespondents(ddmFormAdminDisplayContext.getDDMFormInstance())
		).build()
	%>'
/>
```